deploy/manifests/99_openshift-machineconfig_99-additional-master-ssh.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  creationTimestamp: null
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-additional-master-ssh
```

Consider renaming this to:
```suggestion
  name: 99-assisted-installer-master-ssh
```

---

done